src/discussions/discussions-home/DiscussionsHome.jsx

```diff
@@ -51,7 +51,7 @@ const DiscussionsHome = () => {
   const page = pageParams?.page || null;
   const matchPattern = ALL_ROUTES.find((route) => matchPath({ path: route }, location.pathname));
   const { params } = useMatch(matchPattern);
-  const isOnDesktop = useIsOnDesktop();
+  const isOnTablet = useIsOnTablet();
```

Comment:
use same name for consistency
instead of "isOnTablet" use "isOnTabletorDesktop"